python/bridgestan/model.py

```diff
@@ -13,8 +13,25 @@
 from .compile import compile_model, windows_dll_path_setup
 from .util import validate_readable
 
-FloatArray = npt.NDArray[np.float64]
-double_array = ndpointer(dtype=ctypes.c_double, flags=("C_CONTIGUOUS"))
+
+def array_ptr(*args, **kwargs):
+    """
+    A version of np.ctypeslib.ndpointer
```

Comment:
I prefer the doc for a function to say what the function does.  This is going to return something so I'd phrase it as "Return < > given <...>" wherever possible.

Also use regular punctuation like periods at the end of sentences.

Reply:
I rewrote it, let me know how it reads now